Makefile

```diff
@@ -26,7 +26,12 @@ mlmd-proto:
 	mv ml_metadata/proto/*.go ml_metadata/
 
 metadata-docker-image:
-	docker build -t gcr.io/kubeflow-images-public/metadata .
+	@if test $(shell uname -m) = "aarch64"; then \
+		docker build -t metadata-base-aarch64:latest -f ./dockerfiles/Dockerfile-linux-base.aarch64 . && \
+		docker build -t gcr.io/kubeflow-images-public/metadata --build-arg BASE_IMG=metadata-base-aarch64:latest --build-arg OUTPUT_DIR=linux_arm64_stripped .;\
```

Comment:
Do you have your own docker registry for these images?

`gcr.io` has its own policy about published images but Kubeflow testing infra is on GCP right now which can't test aarch64. Reinforcing `gcr.io` policy on these images will be hard.

You can add links to your registry in the README file if you want.

Reply:
Hi @zhenghuiwang ! Thanks for your kindly suggestions! I am not sure if Arm has its official docker registry and I will confirm with my colleagues now. I am thinking if I use my own docker registry, it would be unprofessional. Therefore I changed all tags for these images to use local registry, and hopefully you are OK with it. If not I will change again. Thanks for your patience.

Reply:
Looks good. Thanks!